schema.graphql

```diff
@@ -27,9 +27,9 @@ input AddressInput {
 type Advertisement {
   _id: ID
   endDate: Date!
-  link: String!
+  mediaUrl: URL!
   name: String!
-  orgId: ID
+  organizationId: ID!
```

Comment:
this should be `organization: Organization`, graphql schema represents relations between objects, the concept of foreign keys(organizationId) is hidden away in the resolvers(query planning) layer 

accordingly create `organization.ts` resolver in `resolvers/Advertisement/` directory to resolve the organization for an advertisement using `parent.organizationId` within that resolver

Reply:
@xoldyckk all the advertisement related resolvers are placed inside the `resolvers/Mutation/` directory. 

Are you suggesting that we should create a `resolvers/Advertisement/` directory and place `createAdvertisement.ts`, `removeAdvertisement.ts`, `updateAdvertisement.ts` and `deleteAdvertisementById.ts` resolvers inside it and also create an `organization.ts` resolver in resolvers/Advertisement/ directory to resolve the organization.

Reply:
@disha1202 read this https://thomasstep.com/blog/writing-graphql-resolvers